test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -638,46 +646,46 @@ object TestAspect extends TimeoutVariants {
 
   /**
    * An aspect that restores the
-   * [[zio.test.environment.TestClock TestClock]]'s state to its starting
+   * [[zio.test.environment.TestClock Has[TestClock]]]'s state to its starting
```

Comment:
Stray changes here and below. Can you make sure you try generating the Scaladoc on this version? I think some of these links are broken.